src/Neo.Extensions/Factories/RandomNumberFactory.cs

```diff
@@ -287,5 +287,19 @@ private static ulong BigMul(ulong a, ulong b, out ulong low)
 
             return ((ulong)ah) * bh + (t >> 32) + (tl >> 32);
         }
+
+        public static byte[] NextBytes(int length, bool cryptography = false)
+        {
+            ArgumentOutOfRangeException.ThrowIfLessThan(length, 1, nameof(length));
```

Comment:
```suggestion
            ArgumentOutOfRangeException.ThrowIfLessThan(length, 0, nameof(length));
```

0 length should not throw